### ISSUE TITLE
`themes.json` should contain the URL of the source file

### ISSUE THREAD

**Issue by julien-f** [Colorsublime/Colorsublime-Themes#188]
IMHO, `theme.json` should have for each theme the URL of the source `.tmTheme` file.

These entries should then be used to regularly keep the themes up to date in this repo.

**Comment by blopker** [comment]
This issue has been brought up before, and it's something we would like to see. There are a few issues though.

1. Validation of updated themes from arbitrary URLs scares the pants off me. This would have to be handled very carefully. 
2. We don't have any server infrastructure aside from GitHub and where ever we host the website. Adding services would require a large lift.

Also, I can only think of one developer ever updating their theme. Updating just doesn't seem to be a big issue right now.

**Comment by julien-f** [comment]
I do not mean that the themes should be served by their URL, just that their repo should be properly referenced and that there should be a script in your repo to fetch the latest versions that you can then review and commit once satisfied.

**Comment by blopker** [comment]
Are you suggesting a script that we run locally at random intervals to pull in updated themes to the repo?

**Comment by julien-f** [comment]
Yep, it's not perfect but it's still better than no updates or than forcing maintainers to create PRs on each update.

**Comment by blopker** [comment]
Sounds like a good compromise (first step even) instead of a automated solution. Not sure when we can get to it, but I'll leave this ticket open. Hopefully some kind soul will submit a PR :)
